### ISSUE TITLE
Add more examples for relationship configurations

### ISSUE THREAD

**Issue by ghost** [dotnet/EntityFramework.Docs#1209]
Please provide an example that's similar to this example: https://github.com/aspnet/EntityFrameworkCore/issues/2517.

Creating a FK without a navigation property is a useful technique in DDD.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 24b2b50e-35f8-9e0b-8ce5-840e1ea816e6
* Version Independent ID: 9e0e6e2c-48fa-76c3-3bd9-813a6df22645
* Content: [Relationships - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/relationships)
* Content Source: [entity-framework/core/modeling/relationships.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/modeling/relationships.md)
* Product: **entity-framework**
* GitHub Login: @rowanmiller
* Microsoft Alias: **divega**

**Comment by Odonno** [comment]
We did it today and we had a hard time to do it (even questioning ourselves if it was even possible). After a couple of hours, we succeed to make a one-to-many relationship without specifying navigation property, only with just 1 foreign key.

And indeed, the syntax is like this: 

```cs
modelBuilder.Entity<Post>()
    .HasOne<Blog>()
    .WithMany()
    .HasForeignKey(p => p.BlogId);
```

We can even specify a lambda for the FK which is really cool.

It definitely need a section in the docs.
